src/modules/DownloadTask.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+
+interface DownloadTaskProps {
+  setButtonText: Function;
+  getData: Function;
+  buttonText: string;
+}
+
+export const DownloadTask = ({
+  setButtonText,
+  getData,
+  buttonText,
+}: DownloadTaskProps) => {
```

Comment:
```suggestion
interface Props {
  setButtonText: Function;
  getData: Function;
  buttonText: string;
}

export const DownloadTask: FC<Props> = ({
  setButtonText,
  getData,
  buttonText,
}) => {
```
это не правильное описание типов, разберись с тем что я написал и исправь подобные ошибки